io.github.sil_car.SqueezeVid.yaml

```diff
@@ -0,0 +1,58 @@
+# https://docs.flatpak.org/en/latest/flatpak-builder-command-reference.html#flatpak-manifest
+app-id: io.github.sil-car.SqueezeVid
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: squeeze-vid
+
+finish-args:
+  # - --env=PYTHONPATH=$PYTHONPATH:/app/lib/python3.10/site-packages/squeeze_vid
+  - --filesystem=home
+
+modules:
+  - name: libsvtav1
+    # Ref: https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/master/Docs/Build-Guide.md#1-build-and-install-svt-av1
+    sources:
+      - type: git
+        url: https://gitlab.com/AOMediaCodec/SVT-AV1.git
+        tag: v1.4.1
+    # source-depth: 1 "shallow clone" is used by default
+    buildsystem: cmake
+    config-opts:
+      # - -G"Unix Makefiles"
+      - -DCMAKE_BUILD_TYPE=Release
+      # - -DCMAKE_INSTALL_PREFIX=/usr
+
+  - name: ffmpeg
+    sources:
+      - type: git
+        url: https://git.videolan.org/git/ffmpeg.git
+        # branch: release/6.0
+        tag: n6.0
+        commit: ea3d24bbe3c58b171e55fe2151fc7ffaca3ab3d2
+    buildsystem: autotools
+    config-opts:
+      # - --prefix=/usr
+      - --disable-doc # removes all docs --disable-{htmlpages,manpages,txtpages}
+      - --disable-ffplay
+      - --enable-version3
+      - --enable-gpl
+      - --enable-libass         # subtitles support
+      - --enable-libaom         # AV1 video encoder
+      # - --enable-libsvtav1      # SVT-AV1 video encoder; requires 22.04+ packages libsvtav1{,-dev}; or build from src
```

Comment:
aren't these build above?

Reply:
Yes. I was initially trying to simplify the build until I got it to work right.